content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md

```diff
@@ -133,7 +134,7 @@ metadata:
 type: Opaque
 ```
 
-### Specifying both `data` and `stringData`
+### Specify both `data` and `stringData`
```

Comment:
As previously mentioned, changes like this can consider whether to add `{#anchor}` for modified headers.

Reply:
My opinion: It's h3 not h2, maybe not that necessary to do it.

Reply:
Yes, only h2 has the anchor currently.